### PR TITLE
Fix 9anime and animesuge

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**
Thanks @theinternet (discord - no idea what his ghub name is) for the decoding code)

### CODE REVIEW

**Comment by nate-moo** [comment]
@theinternetuser

File: anime_downloader/sites/animesuge.py [hunk 1]
```diff
@@ -35,6 +34,81 @@ def _scrape_metadata(self):
 
 
 class AnimeSugeEpisode(AnimeEpisode, sitename='animesuge'):
+    """
+    Decodes the encoded string returned by /ajax/anime/episodes to the embed url.
+
+    Python implementation of decode9AnimeURL, it's basically identical to the js one
+    """ 
+    def decodeString(self, str):
```
Comment (vn-ki) [inline anime_downloader/sites/animesuge.py:42]: If both are the same code, maybe move it util or something?

**Review by vn-ki — COMMENTED** [review]
It's better to keep deobfuscation code close to it's source format, so it's okay I guess.

There isn't much point because they'll break this in a week with some update.

File: anime_downloader/sites/animesuge.py (hunk 1 above)
Comment (AbdullahM0hamed) [reply anime_downloader/sites/animesuge.py]: Made AnimeSugeEpisode inherit NineAnimeEpisode and got rid of the duplicate method

**Pull request merged**